j-ngo828/TackleIt
# Finish backend setup

## What does this PR do?

- Setup backend with dependencies, structure, ts-node-dev, and more
- Add import aliases
- Add ESLint and Prettier

## How should this be tested?

- Run server with `npm run dev` from `./backend`
- Navigate to http://localhost:8080/api/todos, ensure correct text show up with todos count of 20
- Go to `backend/models/todo.ts` change the resolved value to other value than 20. Hit save
- Refresh the localhost page, ensure change is reflected
